### ISSUE TITLE
Pretrain a model on all stock data

### ISSUE THREAD

**Issue by calebheinzman** [Deep-Stonks-Group/Forecasting-Team-1#36]
Train an initial model on all the data then use this model to finely tune to each stock. 

**I estimate the data size will be:**
10 years average per stock.
4,000 stocks
262 days in a year
262 x 4,000 x 10 = 10,480,000 total data points.

Due to the size this will take a lot of training and resources. I can train it on my GPU. 

**Data formatting steps:**
- Data will need to be written to a text file and dynamically read in training.
- Due to this, training will need to be in batches. The model will need to read a bash from the end of the file then train, then move onto the next batch.

**We need to consider how to effectively train on different stocks. Here are a couple of suggestions:**
- Batch train for each stock. Create a dictionary that keeps track of exactly where each stock's data starts and ends. 
- Or shuffle all data points and hope that the model is able to generalize. This doesn't work with traditional normalization methods but it might work with the new relative normalization.

As a part of this issue, we can probably use all 140 technical indicators for training due to the large size of training data. If not create a new card for this.

This model could even be a CNN instead of an LSTM and we can then turn our process into a two step network where it first extracts features using the CNN. then it performs a final classification on the LSTM.
